authbridge/demos/finance-sparc/scripts/setup_keycloak_finance.py

```
@@ -0,0 +1,97 @@
+
+KEYCLOAK_URL = os.environ.get("KEYCLOAK_URL", "http://keycloak.localtest.me:8080")
+KEYCLOAK_REALM = os.environ.get("KEYCLOAK_REALM", "kagenti")
+ADMIN_USER = os.environ.get("KEYCLOAK_ADMIN_USERNAME", "admin")
+ADMIN_PASS = os.environ.get("KEYCLOAK_ADMIN_PASSWORD", "admin")
+UI_CLIENT_ID = os.environ.get("UI_CLIENT_ID", "kagenti")
+SPIFFE_TRUST_DOMAIN = os.environ.get("SPIFFE_TRUST_DOMAIN", "localtest.me")
+NAMESPACE = os.environ.get("NAMESPACE", "team1")
+AGENT_SA = os.environ.get("AGENT_SA", "finance-agent")
+ROPC_CLIENT_ID = os.environ.get("ROPC_CLIENT_ID", "finance-sparc-e2e")
+USER_NAME = "alice"
+USER_PASS = "alice123"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "USER_PASS"

(S105)